### PR TITLE
Identity service implementation

### PR DESCRIPTION
**Pull request by Ishikabaid**
Integrated all the three endpoints

### CODE REVIEW

File: app/controllers/identity.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import ENV from 'website-my/config/environment';
+
+const BASE_URL = ENV.BASE_API_URL;
+
+export default class IdentityController extends Controller {
+  @tracked isEditClicked = false;
+  @tracked isGenerateChaincodeClicked = false;
+  @tracked identityURL = '';
```
Comment (lakshayman) [inline app/controllers/identity.js:11]: Why there are two identityURL?

File: app/controllers/identity.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  @tracked identityURL = this.model.identityURL || '';
+
+  @action async handleRefresh() {
+    window.location.reload();
+  }
+
+  @action handleChaincode() {
+    this.isChaincodeClicked = true;
+    this.Chaincode = 'abcd1234';
+  }
```
Comment (lakshayman) [inline app/controllers/identity.js:26]: I think we can remove this

File: app/controllers/identity.js [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+      try {
+        const response = await fetch(`${BASE_URL}/users/identityURL`, {
+          method: 'PATCH',
+          body: JSON.stringify({ identityURL: this.identityURL }),
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          credentials: 'include',
+        });
+        console.log(response);
```
Comment (lakshayman) [inline app/controllers/identity.js:51]: Remove console.log

**Review by lakshayman — CHANGES_REQUESTED** [review]

File: app/controllers/identity.js (hunk 1 above)
Comment (Ishikabaid) [reply app/controllers/identity.js:11]: Oops, I must have forgotten to remove this one.

File: app/controllers/identity.js (hunk 2 above)
Comment (Ishikabaid) [reply app/controllers/identity.js:26]: Yes

**Review by lakshayman — APPROVED** [review]
LGTM

**Pull request merged**